pkg/api/apisix/client.go

```diff
@@ -114,52 +114,6 @@ func readBody(r io.ReadCloser) (string, error) {
 	return string(data), nil
 }
 
-// getSchema returns the schema of APISIX object.
-func (c *Client) getSchema(ctx context.Context, url string) (string, error) {
```

Comment:
this two functions should be kept

Reply:
those commits have been reverted and the functions have been reverted.